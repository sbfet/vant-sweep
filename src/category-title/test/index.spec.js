import { mount } from '@vue/test-utils';
import CategoryTitle from '../index';

test('render category title', () => {
  const wrapper = mount(CategoryTitle);
  expect(wrapper).toMatchSnapshot();
});
