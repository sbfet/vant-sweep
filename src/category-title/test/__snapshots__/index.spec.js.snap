// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render category title 1`] = `<div class="category-title"></div>`;
